src/components/DatasetCreateView.jsx

```diff
@@ -113,73 +120,152 @@ export class DatasetCreateView extends React.PureComponent {
         </p>
         <Form model="dataset" onSubmit={data => this.handleSubmit(data)}>
           <FormRow>
-            <label htmlFor="dataset.name">Dataset name:</label>
             <Control.text
               model="dataset.name"
               id="dataset.name"
               required
               validators={{ name: this.validateName }}
+              component={(props) =>
+                <TextField
+                  {...props}
```

Comment:
What's going on here? Does this solve the problem that was happening where uploading IDs was resetting the form?